packages/core/test/controls/numericInputTests.tsx

```diff
@@ -57,6 +57,12 @@ describe("<NumericInput>", () => {
             const value = component.state().value;
             expect(value).to.equal("1");
         });
+
+        it("sets allowNumericCharactersOnly to true by default", () => {
+            const component = mount(<NumericInput />);
+            const value = component.props().allowNumericCharactersOnly;
+            expect(value).to.be.true;
```

Comment:
I've come to appreciate unit tests as a way to both verify _and communicate_ every reasonable assumption a consumer can make about a component. Catching bugs and increasing code coverage both play into that, and those are of course the most tangible benefits.

Expanding on the additional piece of my philosophy a bit, if I approach work on an unfamiliar component (e.g.) and see unit tests for Big Behavior X and Big Behavior Y but not Little Behavior Z, then (1) I can't tell if Little Behavior Z is an important part of the contract, (2) I can't always determine how to get the component in that state to verify how Little Behavior Z is supposed to work, and (3) I don't have a safeguard in place to tell me if I break Little Behavior Z. 

For me, testing defaults fits into (3) and also fits into (1) if the default is impactful—but definitely does not fit into (2) (2 is rarer, but I still encounter it occasionally with subtle error-checking code).

Anyway, it's the communication piece that leads me to opt for testing defaults. Happy to go with the majority in this case though; will remove.